### ISSUE TITLE
Automatic light vs dark theme (using the browser's "prefers-color-scheme: dark" CSS mediaqueries)

### ISSUE THREAD

**Issue by nekohayo** [invoiceninja/admin-portal#424]
**What version of Invoice Ninja are you running? ie v4.5 / v5**
v4.5 (but v5 doesn't have this either AFAICS)

**What environment are you running?**
The paid SaaS version

**Have you searched existing issues/requests?**
Yes

**Additional context**

I often use InvoiceNinja late at night, both the time tracker (as a standalone window) and the general features across the admin interface.

I would like the _entire_ interface (not just the menu sidebars) to automatically switch to a dark variant of its stylesheet at nighttime. My operating system (Fedora Linux with GNOME) does the switch for me everyday at the correct times, and my web browser (Firefox) follows suit, telling webpages' CSS mediaqueries that dark is now "preferred" or not.

Note that I don't want a manual dark/light switch toggle in the UI, I would want it to happen automatically "based on browser/OS setting". You can detect this condition using this simple piece of CSS, for example:

`@media (prefers-color-scheme: dark) { blah blah }`

Note that GitHub, as a prime example, already supports this in its UI ;)

**Comment by nekohayo** [comment]
Hi there, I finally upgraded my SaaS account to v5 now, and I was a bit surprised to see it was still not reflecting my operating system's light/dark theme status, unlike other websites. I went into https://app.invoicing.co/#/settings/user_details/preferences and saw a "Dark Mode" manual switch, and no "System" or "Automatic" option for it.

Wasn't this supposed to default to automatically follow the system theme? Is there another place in the UI where I need to be looking to enable that?
